### ISSUE TITLE
can i use this plugin into wechat miniapp ?

### ISSUE THREAD

**Issue by EricHan0723** [surmon-china/vue-awesome-swiper#283]
wechat origin picture preview component  functions are not satisfied my commands, so i want to find some picture plugin use in my project, but wechat miapp can't import jquery or vue plugin, so i wonder if this plugin canbe used to run  :)

**Comment by surmon-china** [comment]
No, because the dom class api provided by wechat miapp is inconsistent with the built-in swiper.
